widget/embedded/src/components/BlockchainsSection/BlockchainsSection.styles.ts

```diff
@@ -3,5 +3,8 @@ import { styled } from '@rango-dev/ui';
 export const Blockchains = styled('div', {
   display: 'grid',
   gap: '$10',
-  gridTemplateColumns: 'repeat(6, minmax(0, 1fr))',
+  gridTemplateColumns: 'repeat(4, 1fr)',
+  '@xs': {
+    gridTemplateColumns: 'repeat(6, 1fr)',
+  },
```

Comment:
Perhaps I am mistaken, but upon reviewing this component with the previous styles, the alignment and size of the blockchain section appeared to be fine. However, with the recent modifications, the blockchain section now seems longer, and the token list appears shorter. Could you please verify if the output matches the image when using the previous styles? If it does, consider reverting your recent changes.
![2](https://github.com/rango-exchange/rango-client/assets/120931880/c10b7d24-5489-48c9-b9a6-19041cbbd0c3)